### DISCUSSION TITLE
Problems with installing the Mac build

### DISCUSSION THREAD

**Discussion by JohnnyMidnite** [nilsschmidt1337/ldparteditor#943] [General]
But I am trying to install the Mac build.  It keeps saying that the app is broken and I should delete it (the download and install work fine).  I have checked the requirements and I don't think that is it.  I have tried it on multiple Macs.  I went to use the apache ant to build an install on my computer and that is where I am totally lost.  I doubt this discussion area is intended for such basic questions... what should I be doing?

**Comment by nilsschmidt1337** [comment]
Hi @JohnnyMidnite,

sorry for the late reply.
When you try to open the app in the normal way, Mac OS X will keep saying that the app is broken.
That is actually a helpful security feature which protects your Mac from untrusted software!

To launch LDPartEditor on Mac you have to open a Terminal and enter `/Applications/LDPartEditor.app/Contents/MacOS/LDPartEditor` to start the App. There is unfortunately no other way to start the App.

Cheers,
Nils


Here is the install description from the release notes:

### Mac OS X
1. Download and extract **LDPartEditor_mac_x64.zip**
2. Mount **LDPartEditor-1.8.65.dmg**
3. Drag **LDPartEditor.app** to the _Applications_ folder
4. Copy **ldparteditor.sh** to your home folder
4a. Open a Terminal.app and run **./ldparteditor.sh**
4b. Or open a Terminal.app and run **/Applications/LDPartEditor.app/Contents/MacOS/LDPartEditor**
